src/ui/views/menu.coffee

```diff
@@ -438,6 +438,10 @@ templateWindow = (viewstate) -> [
         accelerator: getAccelerator('close')
         role: 'close'
     }
+    {
+        label: i18n.__ 'menu.window.toggle_window:Toggle window show'
```

Comment:
Could this also be done via the `Hide YakYak` in `File`? and changing its action to `toggle` *(instead of `hide`)*

 or is there a convention that window menu has this option?

Would `Toggle window show/hide` be a possible string? (I would prefer to change the "Toggle minimize to tray" to something like you suggest here, which could be used by both commands) -- related to #811

Reply:
Probably it can ben done also via the `Hide YakYak` menu but in OSX almost all applications have this (press `CMD + h` for hide an app) convention and I would suggest to not change that but instead add a specific menu item for that feature.

There is no such convention for submenu item in the `Window` menu, I just used that menu because seems appropriate. Of course I need a submenu item (anywhere) to control that feature with a keyboard shortcut in OSX Preferences (see https://goo.gl/wb59Du)

Yes that can be the right string for both the menu item and the system tray. The Italian could be simply changed in `Mostra/Nascondi finestra`

Feel free to change the PR or let me know your preferences I can do that.

Reply:
Thanks for the reply

I've sent a PR request to you that will trickle down to this one. I still haven't figured out if I can contribute to it directly, but this way it's better as it requires your approval :)